sagemaker-pipeline-multi-model/restate-project.ipynb

```diff
@@ -2,31 +2,31 @@
  "cells": [
```

Comment:
<p>please see context -&gt; please see the context</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done as requested, change is in https://github.com/aws/amazon-sagemaker-examples/pull/3488/commits/a90d462d887dd1fe9bd50e908e687acbf42799dd. Thank you!